src/sidebar/components/selection-tabs.js

```diff
@@ -1,49 +1,174 @@
 'use strict';
 
+const { createElement } = require('preact');
+const classnames = require('classnames');
+const propTypes = require('prop-types');
+
 const sessionUtil = require('../util/session-util');
 const uiConstants = require('../ui-constants');
+const useStore = require('../store/use-store');
+const { withServices } = require('../util/service-context');
+
+/**
+ *  Tabbed display of annotations and notes.
+ */
+
+function SelectionTabs({
+  isWaitingToAnchorAnnotations,
+  isLoading,
+  selectedTab,
+  totalAnnotations,
+  totalNotes,
+  totalOrphans,
+  settings,
+  session,
+}) {
+  // actions
+  const store = useStore(store => ({
+    clearSelectedAnnotations: store.clearSelectedAnnotations,
+    selectTab: store.selectTab,
+  }));
+
+  const isThemeClean = settings.theme === 'clean';
+
+  const selectTab = function(type) {
+    store.clearSelectedAnnotations();
+    store.selectTab(type);
+  };
+
+  const showAnnotationsUnavailableMessage = function() {
```

Comment:
This and the two following functions could probably be converted to basic assignment or IIFEs at the least.